capstone/src/main/webapp/script.js

```diff
@@ -13,16 +13,42 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Call the perspective API
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+async function callPerspective(text, lang) {
+  const response = await fetch('/call_perspective', {
```

Comment:
Not something to do in this PR, but we'll want to consider how to handle Perspective failures and edit the code accordingly.

Reply:
Noted!